website/library/reference/how-to-review-someone's-work.md

```diff
@@ -0,0 +1,52 @@
+# How to be a great peer reviewer
+
+When you first start your technical writing journey, one of the many important skills you'll need to learn is how to do is peer reviews. It's a skill that will not only teach you how to collaborate with other technical writers and other people who interact with documentation, but it will also help you to become a better technical writer in the long run.
+
+## Tips for doing effective Peer Reviews
+
+When it comes to peer reviews, it can get confusing and overwhelming at times because you want to be effective, but at the same time you don't want to give too much feedback to point that the TA is confused. Consider using this checklist of things to keep in mind when doing peer reviews:
+
+### Content Review
+
+* **Overall structure and flow**:
+  * Are the steps in the correct order?
+  * Will the user be able to understand the steps?
+  * Have all the terms been defined at the first use?
+* **Accuracy and completeness**:
+  * Are all the steps accurate?
+  * Are there any steps missing or unclear?
+  * If you run through the document from start to finish, does it work?
+
+### Clarity and Readability
+
+* **Wording and clarity**:
+  * Is everything well worded?
+  * Are there any unclear bits?
```

Comment:
"unclear bits" feels like a colloquialism; 'any required details missing" may be more... appropriate?